dags/openshift_nightlies/dag.py

```diff
@@ -194,6 +196,43 @@ def build(self):
     def _get_openshift_installer(self):
         return rogcp.RoGCPInstaller(self.dag, self.config, self.release)
 
+class PrebuiltOpenshiftNightlyDAG(AbstractOpenshiftNightlyDAG):
+    def __init__(self,openshift_release, DagConfig):
+        AbstractOpenshiftNightlyDAG.__init__(self, release=openshift_release, config=DagConfig)
+
+        self.dag = DAG(
+            self.release_name,
+            default_args=self.config.default_args,
+            tags=self.tags,
+            description=f"DAG for Prebuilt Openshift Nightly builds {self.release_name}",
+            schedule_interval=self.config.schedule_interval,
+            max_active_runs=1,
+            catchup=False,
+            params={
+                'KUBEUSER': Param('<Enter openshift cluster-admin username>'),
+                'KUBEPASSWORD': Param('<Enter openshift cluster password>'),
+                'KUBEURL': Param('<Enter cluster URL>')
+            }
+        )
+    
+    def build(self):       
+
+        installer = self._get_openshift_installer()
+        initialize_cluster = installer.initialize_cluster_task()
+
+        with TaskGroup("utils", prefix_group_id=False, dag=self.dag) as utils:
+            utils_tasks = self._get_scale_ci_diagnosis().get_utils()
+            chain(*utils_tasks)
+
+        with TaskGroup("benchmarks", prefix_group_id=False, dag=self.dag) as benchmarks:
+            benchmark_tasks = self._get_e2e_benchmarks().get_benchmarks()
+            chain(*benchmark_tasks)
+
+        initialize_cluster >> benchmarks >> utils
```

Comment:
can you add the platform connector task to this dag now that's been merged?

Reply:
done!